test/Http.test.ts

```diff
@@ -0,0 +1,25 @@
+import { HTTP } from '@src/services/Http';
+import { expect } from 'chai';
+import { stub } from 'sinon';
+import axios from 'axios';
+
+describe('Http', () => {
+	it('Get method timeouts after given time', async () => {
```

Comment:
it would be good if you could add similar test for success (non-timeout) case too

Reply:
Fixed.